js/apps/admin-ui/src/dashboard/Dashboard.tsx

```diff
@@ -138,11 +149,79 @@ const Dashboard = () => {
           data-testid="dashboard-tabs"
           defaultLocation={toDashboard({
             realm,
-            tab: "info",
+            tab: "welcome",
           })}
           isBox
           mountOnEnter
         >
+          <Tab
+            id="welcome"
+            data-testid="welcomeTab"
+            title={<TabTitleText>{t("welcomeTabTitle")}</TabTitleText>}
+            {...welcomeTab}
+          >
+            <PageSection variant="light">
+              <div className="pf-l-grid pf-u-ml-lg">
+                <div className="pf-l-grid__item pf-m-12-col">
+                  <Title
+                    className="pf-u-font-weight-bold"
+                    headingLevel="h2"
+                    size="3xl"
+                  >
+                    {t("welcome")} {realmDisplayInfo}
```

Comment:
The `realmDisplayInfo` should be interpolated into the translation, so that languages that have a different order (such as Arabic, Japanese and Mandarin) can show it properly.